clover/netcdf/variable.py

```diff
@@ -464,6 +464,35 @@ def datetimes(self):
         else:
             return numpy.array([datetime(*d.timetuple()[:6], tzinfo=pytz.UTC) for d in self.dates])
 
+    @property
+    def unit(self):
+        def varys_by_year(x, y):

```

Comment:
Ok, finally figured it out. Git doesn't recognize a line ending change as an actual chance, so it's difficult to commit a line-ending fix. Instead, you have to remove the file from the repo and re-add it:

```
$ git rm --cached <file>
$ git add <file>
```